docs/src/languages/python.md

```diff
@@ -15,142 +16,134 @@ description: Platform.sh supports deploying Python applications. Your applicatio
 
 {{% language-specification type="python" display_name="Python" %}}
 
-## Support libraries
+## Usage example
 
-While it is possible to read the environment directly from your application, it is generally easier and more robust to use the [`platformshconfig`](https://github.com/platformsh/config-reader-python) pip library which handles decoding of service credential information for you.
+### Run your own server
 
-## WSGI-based configuration
+You can define any server to handle requests.
+Once you have it configured, add the following configuration to get it running on Platform.sh:
 
-In this example, we use Gunicorn to run our WSGI application.  Configure the `.platform.app.yaml` file with a few key settings as listed below, a complete example is included at the end of this section.
-
-1. Specify the language of your application (available versions are listed above):
+1. Specify one of the [supported versions](#supported-versions):
 
     {{< readFile file="/registry/images/examples/full/python.app.yaml" highlight="yaml" location=".platform.app.yaml" >}}
 
-2. Build your application with the build hook.
-   Assuming you have your pip dependencies stored in `requirements.txt`
-   and a `setup.py` at [your app root](../create-apps/app-reference.md#root-directory),
-   create a hook like the following:
+2. Install the requirements for your app.
+   For example, to use `pipenv` to manage requirements and a virtual environment, add the following:
+
+   ```yaml {location=".platform.app.yaml"}
+   dependencies:
+       python3:
+           pipenv: "2022.5.2"
```

Comment:
:hammer_and_wrench: ⏯️ 
We could add a variable for that one or [retrieve it directly](https://pypi.org/project/pipenv/#history), especially since this one is re-used throughout the page.

Reply:
Retrieving it would mean writing a script to scrape the page/feed, which seems perhaps excessive. :shrug: Can do a shortcut for the repeated use of the same snippet.